app/services/customers/base_service.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Customers
+  class BaseService < BaseService
```

Comment:
Rather than adding a level of inheritance, what about adding a concern module to share the behavior between multiple service class?

Something like `Customers::PaymentProviderFinder`?